cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -44,14 +46,10 @@ consists of four mandatory roles and two optional roles:
 
 These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of the same name.
 
-- Note that the *CDAP Gateway/Router Service* is not a `Cloudera Manager Gateway Role 
-  <http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cm_mc_managing_roles.html>`__
-  but is instead another name for the CDAP Router Service.
-
 - As CDAP depends on HDFS, YARN, HBase, ZooKeeper, and (optionally) Hive and Spark, it must be placed
```

Comment:
i see below in other sections it's "must be installed on cluster host(s)" which is a little better.